msteams-platform/m365-apps/publish.md

```diff
@@ -13,13 +13,13 @@ ms.subservice: m365apps
 Microsoft Teams supports Microsoft 365-enabled Teams apps for production. You can distribute these apps to users who use:
 
 * *Targeted Release* (dev preview) versions of Outlook.com and microsoft365.com (formerly office.com)
-* *Beta Channels* build of Outlook for Windows desktop, Outlook for Android, and Microsoft 365 Current Channel (dev preview) build of Microsoft 365 app for Android
+* *Beta Channels* build of Outlook for Windows desktop, Outlook for Android, and Microsoft 365 Current Channel build of Microsoft 365 app for Android
```

Comment:
Let's rework this:

- Web: *Current Channel* versions of outlook.com and microsoft365.com (formerly *office.com*)
- Desktop: *Current Channel* builds of Outlook and Microsoft 365 (Office) app for Windows desktop
- Mobile: Production builds of Microsoft 365 (Office) app on iOS and Android